ecs-cli/modules/utils/compose/convert_task_definition_test.go

```diff
@@ -640,7 +731,7 @@ func TestMemReservationHigherThanMemLimit(t *testing.T) {
 		EnvironmentLookup: envLookup,
 		ResourceLookup:    resourceLookup,
 	}
-	_, err = ConvertToTaskDefinition(taskDefName, context, serviceConfigs, "")
+	_, err = ConvertToTaskDefinition(taskDefName, context, serviceConfigs, "", "")
 	assert.EqualError(t, err, "mem_limit should not be less than mem_reservation")
```

Comment:
Why is this the expected error?

Reply:
Not sure, but that's not part of my change --I was just modifying the function call since the method signature changed.

Reply:
I said this in person as well, but I don't like the idea asserting on the exact content of an error message.